src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8394,11 +8420,13 @@ private void VerifyQuotedInvalid<T>(string name, string value, T expected, Func<
             [Fact]
```

Comment:
Can we add a test which validates that on Linux we are choosing the correct PDB format?